packages/account-tree-controller/src/AccountTreeController.ts

```diff
@@ -724,6 +734,14 @@ export class AccountTreeController extends BaseController<
       state.accountTree.selectedAccountGroup = groupId;
     });
 
+    this.messagingSystem.publish(
+      `${controllerName}:selectedAccountGroupUpdated`,
```

Comment:
WDYT about using `:selectedAccountGroupChange` instead? This would match the `:selectedAccountChange` we already have on the accounts-controller?